src/regex/bytes.rs

```diff
@@ -1665,6 +1665,24 @@ impl<'h> Captures<'h> {
             .map(|sp| Match::new(self.haystack, sp.start, sp.end))
     }
 
+    /// Return the overall match for the capture.
+    ///
+    /// This returns the match for index `0`. That is it is equivalent to
+    /// `m.get(0).unwrap()`
+    ///
+    /// ```
+    /// use regex::bytes::Regex;
+    ///
+    /// let re = Regex::new(r"[a-z]+([0-9]+)").unwrap();
+    /// let caps = re.captures(b"   abc123-def").unwrap();
+    ///
+    /// assert_eq!(caps.as_match().as_bytes(), b"abc123");
+    /// ```
```

Comment:
This example is not formatted in the same way as all other examples.

Please look at other public API items and format this one in the same way. It should have an `# Example` heading, for example.